src/victory-primitives/bar.js

```diff
@@ -81,7 +81,8 @@ export default class Bar extends React.Component {
   getBarWidth(props, style) {
     const {width, data} = props;
     const padding = props.padding.left || props.padding;
-    const defaultWidth = data.length === 0 ? 8 : (width - 2 * padding) / data.length;
+    const length = Array.isArray(data) ? data.length : 0;
```

Comment:
@DerekMaffett the `null` data is coming back when users return `null` from an event mutation. My guess is that because we are using  `_.defaults` to combine `baseProps` with event state, so `undefined` is being handled correctly, but `null` is not. That said, this error was not showing up when returning null prior to my changes related to shouldComponentUpdate in the primitive components, and the change is also not showing up in shared events. My guess for a systematic fix would be to add null checking to `add-events`.